Source/AliveLibAO/Slig.cpp

```diff
@@ -490,7 +495,162 @@ void Slig::VScreenChanged_465480()
+    {
+        field_210_resources.res[7] = ResourceManager::GetLoadedResource_4554F0(ResourceManager::Resource_Animation, 419, 1, 0);
+    }
+    if (!field_174_tlv.field_50_disable_resources.Get(SligFlags_DisabledRes::eDisabledRes_Bit2))
+    {
+        field_210_resources.res[8] = ResourceManager::GetLoadedResource_4554F0(ResourceManager::Resource_Animation, 420, 1, 0);
+    }
+    if (!field_174_tlv.field_50_disable_resources.Get(SligFlags_DisabledRes::eDisabledRes_Bit10))
+    {
+        field_210_resources.res[4] = ResourceManager::GetLoadedResource_4554F0(ResourceManager::Resource_Animation, 416, 1, 0);
+    }
+    if (!field_174_tlv.field_50_disable_resources.Get(SligFlags_DisabledRes::eDisabledRes_Bit11))
+    {
+        field_210_resources.res[9] = ResourceManager::GetLoadedResource_4554F0(ResourceManager::Resource_Animation, 426, 1, 0);
+    }
+    if (!field_174_tlv.field_50_disable_resources.Get(SligFlags_DisabledRes::eDisabledRes_Bit3))
+    {
+        field_210_resources.res[5] = ResourceManager::GetLoadedResource_4554F0(ResourceManager::Resource_Animation, 417, 1, 0);
+    }
+    if (!field_174_tlv.field_50_disable_resources.Get(SligFlags_DisabledRes::eDisabledRes_Bit7))
+    {
+        field_210_resources.res[1] = ResourceManager::GetLoadedResource_4554F0(ResourceManager::Resource_Animation, 413, 1, 0);
+    }
+    if (!field_174_tlv.field_50_disable_resources.Get(SligFlags_DisabledRes::eDisabledRes_Bit4))
+    {
+        field_210_resources.res[13] = ResourceManager::GetLoadedResource_4554F0(ResourceManager::Resource_Animation, 344, 1, 0);
+    }
+    if (!field_174_tlv.field_50_disable_resources.Get(SligFlags_DisabledRes::eDisabledRes_Bit6))
+    {
+        field_210_resources.res[14] = ResourceManager::GetLoadedResource_4554F0(ResourceManager::Resource_Animation, 333, 1, 0);
+    }
+    if (!field_174_tlv.field_50_disable_resources.Get(SligFlags_DisabledRes::eDisabledRes_Bit5))
+    {
+        field_210_resources.res[15] = ResourceManager::GetLoadedResource_4554F0(ResourceManager::Resource_Animation, 28, 1, 0);
+    }
+    field_11C = gMap_507BA8.field_4_current_camera;
+    field_20E_spotted_possessed_slig = 0;
+    field_114_timer = gnFrameCount_507670 + field_174_tlv.field_1C_pause_time;
+    switch (field_174_tlv.field_1A_start_state)
+    {
+        case Path_Slig::StartState::Paused_1:
+        {
+            field_1F8_fn = &Slig::Brain_Inactive_46B780;
+            field_1FC_fn2 = &Slig::Brain_Inactive_46B780;
+            break;
+        }
+        case Path_Slig::StartState::Sleeping_2:
+        {
+            if (field_174_tlv.field_1_unknown && field_174_tlv.field_4E_stay_awake)
+            {
+                field_1F8_fn = &Slig::Brain_Inactive_46B780;
+                field_1FC_fn2 = &Slig::Brain_Inactive_46B780;
+            }
+            else
+            {
+                field_1F8_fn = &Slig::Brain_Sleeping_46B4E0;
```

Comment:
you need to use SetBrain on these else code that compares the function pointer addresses will blow up